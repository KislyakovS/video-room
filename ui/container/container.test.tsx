import { render } from '@testing-library/react';

import Container from './container';

describe('Component Component:', () => {
    it('snapshot', () => {
        const { container } = render(<Container>Container</Container>);

        expect(container).toMatchSnapshot();
    });
})