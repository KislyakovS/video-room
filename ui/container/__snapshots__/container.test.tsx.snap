// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Component: snapshot 1`] = `
<div>
  <div
    class="container"
  >
    Container
  </div>
</div>
`;
